phobos/blender/model/models.py

```diff
@@ -1014,21 +1014,21 @@ def deriveModelDictionary(root, name='', objectlist=[]):
 
     modeldescription = bUtils.readTextFile('README.md')
 
-    model = {
-        'links': {},
-        'joints': {},
-        'sensors': {},
-        'motors': {},
-        'controllers': {},
-        'materials': {},
-        'meshes': {},
-        'lights': {},
-        'groups': {},
-        'chains': {},
-        'date': datetime.now().strftime("%Y%m%d_%H:%M"),
-        'name': modelname,
-        'version': modelversion,
-        'description': modeldescription,
+    model = {  # Used in:
+        'links': {},  # representation.robot
+        'joints': {},  # representation.robot
+        'sensors': {},  # smurf.robot.Smurf
+        'motors': {},  # smurf
```

Comment:
If you leave those comments, clean them up, please. e.g. smurf.motors

Reply:
I missed these